rules/linux/persistence_linux_user_added_to_privileged_group.toml

```
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2023/02/13"
+integration = ["system"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/04/03"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to add a user to a privileged group. Attackers may add users to a privileged group in order to 
+escalate privileges or establish persistence on a system or domain.
+"""
+from = "now-9m"
+index = ["logs-system.auth-*", "logs-endpoint.events.*"]
```

```suggestion
index = ["logs-endpoint.events.*"]
```
Consider testing this one on endgame and adding it